mongo.go

```diff
@@ -260,9 +260,12 @@ func (m Handler) Find(ctx context.Context, q *query.Query) (*resource.ItemList,
 	// If the number of returned elements is lower than requested limit, or not
 	// limit is requested, we can deduce the total number of element for free.
 	if limit < 0 || len(list.Items) < limit {
-		list.Total = len(list.Items)
 		if q.Window != nil && q.Window.Offset > 0 {
-			list.Total += q.Window.Offset
+			if len(list.Items) > 0 {
+				list.Total = q.Window.Offset + len(list.Items)
```

Comment:
There is one case case is not covered: when the total is equal the offset. In this case the number of items will be 0.

Reply:
It is, just it's handled the same way as greater than, by not setting total, which seams to be correct. Maybe add a test case for it though

Reply:
Done, please review.